src/toil/wdl/toilwdl.py

```diff
@@ -17,21 +17,21 @@
 import subprocess
 import sys
 
-import toil.wdl.wdl_parser as wdl_parser
-from toil.wdl.wdl_analysis import AnalyzeWDL
-from toil.wdl.wdl_synthesis import SynthesizeWDL, write_AST
+from toil.wdl import dict_from_JSON
+from toil.wdl.versions.draft2 import AnalyzeDraft2WDL
+from toil.wdl.wdl_synthesis import SynthesizeWDL
 
-wdllogger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
 
 
 def main():
-    '''
+    """
     A program to run WDL input files using native Toil scripts.
 
     Calls two files, described below, wdl_analysis.py and wdl_synthesis.py:
 
-    wdl_analysis reads the wdl, json, and extraneous files and restructures them
-    into 2 intermediate data structures before writing (python dictionaries):
+    wdl_analysis reads the wdl and restructures them into 2 intermediate data
```

Comment:
Extraneous `tsv` files now break, but `json` is still correct, right?

Reply:
Good catch. I just moved that part to the `wdl_synthesis` section below since JSON files were never used in the analysis part. 

I also updated the docs in 3d116586afe756fe4454ee3e3cbc19fbf92f8e77 so that needs a review as well. Thanks!